lib/android/app/src/main/java/com/reactnativenavigation/utils/ImageUtils.java

```diff
@@ -0,0 +1,39 @@
+package com.reactnativenavigation.utils;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.BitmapDrawable;
+import android.graphics.drawable.Drawable;
+import android.os.StrictMode;
+import android.support.annotation.NonNull;
+
+import java.io.IOException;
+import java.net.URL;
+
+public class ImageUtils {
+    @NonNull
+    public static Drawable tryLoadIcon(Context currentContext, String uri) {
+
+        StrictMode.ThreadPolicy threadPolicy = StrictMode.getThreadPolicy();
```

Comment:
why do you need it?

Reply:
I got this from V1.  Basically, it allows us to "download" the image synchronously instead of async.

Reply:
I don't see any reasons doing it on `main` thread. It's rather better to do it async.